### DISCUSSION TITLE
Multiple flow line data sources as VBET evidence?

### DISCUSSION THREAD

**Discussion by philipbaileynar** [Riverscapes/riverscapes-tools#142] [VBET] [answered]
There's nothing to say that we must only use one set of flow lines when calculating channel evidence!

We currently rasterize the NHDPlusHR (1:24K) flow lines, but we already know that their alignment is problematic in certain places.

We could also rasterize the NHDPlus (1:100K) and combine them with the rasterized 1:24K lines. Obviously the 1:100K does not extend as far, and technically they are intended for a different resolution, but they will overlap in a lot of places and be close in others. We could combine the two so that the overlaps boost the likelihood of being VB.

I guess at a high level, I'm looking for other nationally available datasets that are easy to throw on the pile without making VBET hard to explain because there are too many lines of evidence in the mix.

@joewheaton @MattReimer @KellyMWhitehead ?

**Comment by joewheaton** [comment]
Oh now this is really, really interesting.  I like this and when you combine it with flow areas it is pretty useful.  One thing that Will Graf developed based on mapping channels through time from imagery was a locational probability:
![](https://ars.els-cdn.com/content/image/1-s2.0-S0169555X04001126-gr6.jpg)

based on these traces... 

Graf WL. 2000. Locational Probability for a Dammed, Urbanizing Stream: Salt River, Arizona, USA. Environmental Management 25 : 321–335.

We could use the same logic here. Real simple. and Elegant. @shelbysawyer ... thoughts?

**Comment by joewheaton** [comment, marked as answer]
We have our hands full for now with where to go with VBET. @shelbysawyer I am not sure you ever read this. Please do. @philipbaileynar we don't need more evidence, we need to use what we've got and get VBET working in an understood way. I'm saying this is "Answered" for now and we can always come back to this discussion at some point in future if interesting.

**Reply by shelbysawyer** [reply]
@joewheaton This must have come in before I figured out the notification glitch. Really interesting-- sorry to have missed this back in January.

**Reply by joewheaton** [reply]
This idea of other lines of evidence is shelved for now. However, the idea of flow-lines has been converted into one of flow-areas.
